static/openapi/prerendering-spec.json

```diff
@@ -1997,21 +1708,142 @@
+                          "type": "string",
+                          "description": "Interaction unique identifier."
+                        }
+                      },
+                      "required": ["_key"]
+                    }
+                  },
+                  "required": ["status", "data"],
+                  "example": {
+                    "status": "OK",
+                    "message": "Interaction updated successfully",
+                    "data": { "_key": "-A5HO1NwgFrTMp5xv8KRWnk" }
+                  }
+                }
+              }
+            }
+          },
+          "400": {
+            "description": "Bad request.",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "type": "object",
+                  "description": "Bad request.",
+                  "properties": {
+                    "status": {
+                      "type": "string",
+                      "description": "Error status."
+                    },
+                    "message": {
+                      "type": "string",
+                      "description": "Error message."
+                    }
+                  },
+                  "required": ["status", "message"],
+                  "example": {
+                    "status": "ERROR",
+                    "message": "Header x-photoniq-customerid is required"
+                  }
+                }
+              }
+            }
+          },
+          "500": {
+            "description": "Internal server error.",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "type": "object",
+                  "description": "Internal server error.",
+                  "properties": {
+                    "status": {
+                      "type": "string",
+                      "description": "Error status."
                     },
-                    "required": ["config"]
+                    "message": {
+                      "type": "string",
+                      "description": "Error message."
+                    }
+                  },
+                  "required": ["status", "message"],
+                  "example": {
+                    "status": "ERROR",
+                    "message": "Internal server error"
                   }
-                ]
+                }
               }
             }
           }
-        },
+        }
+      },
+      "delete": {
+        "summary": "Delete synthetic interaction",
+        "tags": ["Synthetic Interactions"],
+        "description": "Deletes a synthetic interaction associated with the origin.",
         "parameters": [
           {
             "schema": { "type": "string", "minLength": 1 },
             "in": "path",
             "name": "origin",
             "required": true,
-            "description": "Origin hostname"
+            "description": "Origin hostname."
```

Comment:
When the parameter is part of the path, it is required to have `minLength`, otherwise Fastify does not recognize it as required (I don't know why). If it is not part of the path, then `required: true` is enought.